png_benchmark.py

```diff
@@ -0,0 +1,144 @@
+import gc
+import io
+import json
+import os
+import pathlib
+import time
+
+from PIL import Image
+
+# Setting parameters
+FRAME_COUNT = 10  # Total number of frames
+FRAME_TO_SAVE = 69  # This frame is good, it has both landing pads in it
+INPUT_PATH = pathlib.Path("test_images", "Encode Test Dataset 2024")
+OUTPUT_PATH = pathlib.Path(f"log_{int(time.time())}")
+COMPRESS_TYPES = [0, 1, 2, 3, 4]  # There are 5 different compression algorithms, each are numbered
+INITIAL_COMPRESS_LEVEL = 1  # Compress level 0 is skipped because it is uncompressed
+MAX_COMPRESS_LEVEL = 6  # Although this maxes out at 9, it takes way too long (like 10s per image)
+
+# Keys for dictionary entries
+MAX_TIME_MS = "max_time_ms"
+MIN_TIME_MS = "min_time_ms"
+AVG_TIME_MS = "avg_time_ms"
+MAX_SIZE_B = "max_size_B"
+MIN_SIZE_B = "min_size_B"
+AVG_SIZE_B = "avg_size_B"
+MAX_COMPRESSION = "max_size_ratio_%"
+MIN_COMPRESSION = "min_size_ratio_%"
+AVG_COMPRESSION = "avg_size_ratio_%"
```

Comment:
Make this explicit in variable naming and text key that it is the old size that is considered 100% and a smaller file will result in a smaller value. The words compression and ratio by themselves do not currently indicate this.